### DISCUSSION TITLE
Welcome to esp-csi Discussions!

### DISCUSSION THREAD

**Discussion by zhanzhaocheng** [espressif/esp-csi#118] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by onefishfish** [comment]
esp-csi  win10 开发环境 idf5.3 无法编译，一般什么情况，有无教程，想测试一下

**Reply by NingYuxiang** [reply]
可以参考文档以部署idf环境：https://docs.espressif.com/projects/esp-idf/zh_CN/release-v5.3/esp32s3/get-started/index.html

**Comment by onefishfish** [comment]
idf5.3.1下 环境 无法编译，有安装的教程实例吗

**Comment by onefishfish** [comment]
esp32  烧录后，输出总是提示  ：Hello from app_main!
Hello from app_main!
Hello from app_main!
Hello from app_main!
Hello from app_main!
Hello from app_main!
Hello from app_main!
Hello from app_main!
Hello from app_main!
Hello from app_main!

**Reply by NingYuxiang** [reply]
请确认当前的工程里是否有输出这样log的代码，或者有可能烧录错程序了，请确认当前的工程路径和烧录程序的路径是否相同

**Comment by onefishfish** [comment]
esp32 s3板子 无法添加到 ESP RainMaker应用程序  ，不知道问题出在哪里

**Reply by NingYuxiang** [reply]
请参考 https://github.com/espressif/esp-rainmaker 中的例程以实现Rainmaker的部署，如果您仍然遇到问题，请提供错误的log以供分析

**Comment by onefishfish** [comment]
![3d0508f8109b297ebf86c776ad49776](https://github.com/user-attachments/assets/23e67624-ee7f-467c-8f3f-809b0b918558)
zhe这些提示

**Comment by onefishfish** [comment]
烧录 出现这些提示 什么原因  
![3900f32797378473de86074fbb36d0f](https://github.com/user-attachments/assets/4bf8bd4b-10b1-4342-b255-cf55b13ab19b)

**Reply by NingYuxiang** [reply]
您可以暂时参照 https://github.com/espressif/esp-csi/pull/223/files 进行修改，并执行 
```c
idf.py fullclean
rm -rf build
```
之后重新编译运行

**Comment by onefishfish** [comment]
![7873e976e39a23609fb4d93bbfa9aac](https://github.com/user-attachments/assets/7f1deabf-4dfb-4221-81c3-75672165fcee)
这些也能正常 ，不知道出在哪里问题

**Comment by onefishfish** [comment]
idf.py build 运行后  在build文件夹也能看到 3个 bin 文件，但是总是报 tiem_t now 错误
